pwa/src/templates/templateParts/resultsTemplates/table/TableResultTemplate.tsx

```diff
@@ -56,21 +56,31 @@ export const TableResultTemplate: React.FC<LayersResultTemplateProps> = ({ compo
               <TableCell>
                 <div
                   className={clsx(
-                    styles[_.camelCase(t(`${component.nl?.commonground?.layerType ?? "Unknown"} layer`))],
+                    styles[
+                      _.camelCase(
+                        t(
+                          `${
+                            (component._self.schema.ref.includes("component.schema.json") &&
+                              component.embedded?.nl?.embedded?.commonground?.layerType) ??
+                            "Unknown"
+                          } layer`,
+                        ),
+                      )
+                    ],
```

Comment:
I think this is getting too complex to be _within_ the TSX return statement. 

I'd opt to map the components _before_ getting to the return statement.